modules/schema/pages/competition-results.adoc

```diff
@@ -982,6 +990,7 @@ An XML instance excerpt is provided below for illustration:
     <efac:LotTender>
         <cbc:ID schemeName="tender">TEN-0001</cbc:ID>
         <cbc:RankCode>1</cbc:RankCode>
+        <efbc:ForeignSubsidiesMeasuresCode listName=”foreign-subsidies-measure-conclusion”>adm-clos</efbc:ForeignSubsidiesMeasuresCode >
```

Comment:
The double quotes here are the fancy Unicode quotation marks, and not the regular ".
Someone copy-pasting from here into an XML might get confused because the resulting XML will be invalid.